### PR TITLE
Add notice about stopping support for v1.21

### LINKED ISSUE #1667: Add notice about 1.21 end of support

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*
closes #1667

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kschumy** [comment]
/hold
waiting on timing clarification

File: README.md [hunk 1]
```diff
@@ -67,6 +58,16 @@ Source: https://github.com/cncf/artwork/tree/master/projects/kubernetes/certifie
 | --- | --- | --- |
 | 24 | [v1-21-eks-24](https://distro.eks.amazonaws.com/kubernetes-1-21/kubernetes-1-21-eks-24.yaml) | [v1.21.14](https://github.com/kubernetes/kubernetes/release/tag/v1.21.14) |
 
+**🚨 IMPORTANT INFORMATION ABOUT v1.21 SUPPORT 🚨**
+
+Aligning with the [Amazon EKS release calendar](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar),
```
Comment (markapruett) [inline README.md:63]:
```suggestion
In alignment with the [Amazon EKS release calendar](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar), 
```

Comment (markapruett) [reply README.md:63]: To ensure it doesn't sound like we're just doing this.

File: README.md [hunk 2]
```diff
@@ -67,6 +58,16 @@ Source: https://github.com/cncf/artwork/tree/master/projects/kubernetes/certifie
 | --- | --- | --- |
 | 24 | [v1-21-eks-24](https://distro.eks.amazonaws.com/kubernetes-1-21/kubernetes-1-21-eks-24.yaml) | [v1.21.14](https://github.com/kubernetes/kubernetes/release/tag/v1.21.14) |
 
+**🚨 IMPORTANT INFORMATION ABOUT v1.21 SUPPORT 🚨**
+
+Aligning with the [Amazon EKS release calendar](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar), 
+EKS Distro will be discontinuing support of Kubernetes v1.21 soon. 
+
+While there are no plans to removed EKS-D v1.21 images from the [ECR](https://gallery.ecr.aws/eks-distro),
```
Comment (markapruett) [inline README.md:66]:
```suggestion
While there are no plans to remove EKS-D v1.21 images from the [ECR](https://gallery.ecr.aws/eks-distro),
```

File: README.md [hunk 3]
```diff
@@ -75,7 +76,7 @@ no plans to remove these versions' images from EKS Distro ECR, there will be no
 more updates, including security fixes, for them.
 
 **Due to the increased security risk this poses, it is HIGHLY recommended that
-users of v1.18 - v1.20 update to a supported version (v1.21+) as soon as
+users of v1.18 - v1.20 update to a supported version (v1.22+) as soon as
```
Comment (markapruett) [inline README.md:79]:
```suggestion
users of v1.18 - v1.21 update to a supported version (v1.22+) as soon as
```

Comment (markapruett) [reply README.md:79]: Should this be 1.21 instead?

File: README.md (hunk 2 above)
Comment (kschumy) [reply README.md:66]: added

**Review by markapruett — APPROVED** [review]

**Pull request merged**